.devcontainer/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could you please use our standard license header

https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/contributing.md#file-headers

Reply:
edited